### ISSUE TITLE
Unable to make requests from .md file

### ISSUE THREAD

**Issue by backpackcoder** [Huachao/vscode-restclient#53]
- VSCode Version:
Version 1.7.2
Commit 7ba55c5860b152d999dda59393ca3ebeb1b5c85f
Date 2016-11-21T22:14:18.217Z
Shell 1.3.8
Renderer 52.0.2743.82
Node 6.5.0

- OS Version:
Ubuntu 16.04

- REST Client Version:
v0.11.0

I'm unable to put sample requests in my README.md file to document an API.  Here is a small sample of my README.md file:
```
...

Get a list of all users
-----------------------

GET http://localhost:5000/api/users HTTP/1.1


Create a new user
--------------------

POST http://localhost:5000/api/users HTTP/1.1
content-type: application/json

{
    "name": "Test User"
}

...
```
Steps to Reproduce:

1.  If I select one of the requests and hit ctrl-alt-r nothing happens.
2.  If I change the file extension to .txt and repeat step 1. The request is made and the response panel opens.
3. If I change the file extension back to .md and repeat step 1. nothing happens again.

**Comment by Huachao** [comment]
@backpackcoder this is by design, we currently only allows triggered for file type of http(.http or .rest) or file type of text(.txt)

**Comment by backpackcoder** [comment]
Thanks for the response.  I'm sorry if I missed that in the docs as it was late last night when I submitted the ticket.  Feel free to close.

**Comment by Huachao** [comment]
@backpackcoder thanks for your usage, and it's my fault that didn't express it clearly in docs, I will update it. 😄

**Comment by Huachao** [comment]
@acuarica @backpackcoder this has been implemented in the latest version 0.25.0 with the contribution from @acuarica
